

const fetchUser = async () => {
    try {
        
    } catch (error) {
        
    }
    
}